scripts/prepare-release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+PREVIOUS_VERSION="$(cat deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | grep integreatly-operator | awk -F v '{print $2}')"
+
+create_new_csv() {
+  operator-sdk generate csv --csv-version "$VERSION" --default-channel --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
```

Comment:
need to explicitly specify the operator name here using the `--operator-name` flag. otherwise the name of the directory will be used as the operator name.